unitTests/magicbox/ResultPreviewsManagerTest.ts

```diff
@@ -0,0 +1,123 @@
+import { ResultPreviewsManager, ISearchResultPreview } from '../../src/magicbox/ResultPreviewsManager';
+import { IMockEnvironment, MockEnvironmentBuilder } from '../MockEnvironment';
+import { $$ } from '../../src/utils/Dom';
+import { ResultPreviewsManagerEvents, IPopulateSearchResultPreviewsEventArgs } from '../../src/events/ResultPreviewsManagerEvents';
+import { toArray } from 'lodash';
+
+export function ResultPreviewsManagerTest() {
+  describe('ResultPreviewsManager', function() {
+    let env: IMockEnvironment;
+    function buildEnvironment() {
+      return (env = new MockEnvironmentBuilder().build());
+    }
+
+    let container: HTMLElement;
+    function buildContainer() {
+      return buildEnvironment().root.appendChild((container = $$('div').el));
```

Comment:
Rather than having `container`, could we retrieve it by accessing the correct property from the returned environment object?

Similar comment for the other lines in this test file like 26, 33 etc.

Reply:
Good point, I didn't know `buildEnvironment` already had the `element` property, I'm going to use that instead.